package.json

```diff
@@ -38,15 +53,20 @@
     "express": "^4.17.1",
     "figlet": "^1.2.4",
     "helmet": "^3.21.2",
+    "jest": "^24.9.0",
     "jsonfile": "^5.0.0",
     "microframework": "^0.6.4",
+    "microframework-w3tec": "^0.6.3",
```

Comment:
Remove this dependency

Reply:
@VivekFitkariwala Removed the unnecessary dependency from the code.